<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>33</y>
     <width>811</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="LoadImageButton">
        <property name="text">
         <string>Load Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ColorSeperationButton">
        <property name="text">
         <string>Color Seperation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ColorTransButton">
        <property name="text">
         <string>Color Transformations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BlendButton">
        <property name="text">
         <string>Blending</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="GaussianBlurButton">
        <property name="text">
         <string>Gaussian Blur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BilateralButton">
        <property name="text">
         <string>Bilateral filter </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MedianButton">
        <property name="text">
         <string>Median filter </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="EdgeGaussianBlurButton">
        <property name="text">
         <string>Gaussian Blur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SobelXButton">
        <property name="text">
         <string>Sobel X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SobelYButton">
        <property name="text">
         <string>Sobel Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MagnitudeButton">
        <property name="text">
         <string>Magnitude</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="ResizeButton">
        <property name="text">
         <string>Resize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TranslationButton">
        <property name="text">
         <string>Translation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RotationScalButton">
        <property name="text">
         <string>Rotation, Scaling</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ShearingButton">
        <property name="text">
         <string>Shearing</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
